### ISSUE TITLE
you can get the logits from `.generate()` afaik

### ISSUE THREAD

**Issue by Jemoka** [alliegriffith/perez_reimplementation#12]
https://stackoverflow.com/questions/71376760/how-to-output-the-list-of-probabilities-on-each-token-via-model-generate

saves half of the LM calls :)

https://github.com/alliegriffith/perez_reimplementation/blob/e36436c3bc2e6664b79564a31dc5ecf2f55ae3c7/perez.py#L205

**Comment by Jemoka** [comment]
oh but **idt they have gradients attached**, which is bad; check this
